EjerciciosT15/src/paquetesiete/Principal.java

```diff
@@ -16,8 +16,8 @@
      */
     public static void main(String[] args) {
         // 
-        double sueldosSucursal1 = 0;
-        double sueldosSucursal2 = 0;
+        double sueldosSucursal1 = LecturaArchivo.leerRegistros();
+        double sueldosSucursal2 = LecturaArchivo.leerRegistrosDos();
         System.out.printf("La suma total de sueldos es %.2f\n", 
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.